plugins/azure/mediaServices/amsClassicApiDisabled.js

```diff
@@ -0,0 +1,59 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Media Services Classic API Disabled',
+    category: 'Media Services',
+    domain: 'Content Delivery',
+    description: 'Ensure that Microsoft Azure Media Services do not have the Classic API enabled.',
+    more_info: 'Disabling the Classic API for Azure Media Services is recommended to utilize modern APIs and features. Enabling classic features can enable the use of classic V2 APIs but might disable advanced security features like managed identities.',
+    link: 'https://learn.microsoft.com/en-us/azure/media-services/latest/migrate-v-2-v-3-differences-api-access',
```

Comment:
i cannot find the keyword classic referenced with Media Service in this link? Can you add a link that defines what classic apis are and why we should not use them?